lib/pure/asyncftpclient.nim

```diff
@@ -79,7 +79,12 @@ proc connect*(ftp: AsyncFtpClient) {.async.} =
     # 120 Service ready in nnn minutes.
     # We wait until we receive 220.
     reply = await ftp.expectReply()
+
+  # Handle 220 messages from the server
   assertReply(reply, "220")
+  while reply.continuesWith("-", 3): # handle multiline 220 message
```

Comment:
What if the server continues to send ``220-``? It will block the app forever. Add a limit for this please.

Also, use ``reply[3] == '-'`` instead of ``continuesWith``